simple/stats/config.md

```diff
@@ -17,6 +17,16 @@ The config parameters for the files to be imported should be specified in a `con
     "geoid.csv": {
       "entityType": ""
     }
+  },
+  "variables": {
```

Comment:
Can we also handle the case when variable does not have "group" field, then it is linked under "dc/g/Root" directly.

Reply:
Done.